metadata/cadastre/utah-parcels

```diff
@@ -0,0 +1,75 @@
+# Title
+
+Utah Basic Parcels
+
+## ID
+
+cbe4da2a-3edb-4a4a-b082-60f9371c8006
+
+## Brief Summary
+
+Parcel boundaries for all 29 counties in Utah that includes basic information such as parcel address and generalized land ownership.
+
+## Summary
+
+Utah Basic Parcels contains information such as the county name, parcel address, land owner type, and phone number for the county recorder. Landowners of privately owned parcels are not specified.
+
+## Description
+
+### What is the dataset?
+
+A parcel is a defined area of land that is owned by a public or private entity. Utah Basic Parcels includes a polygon layer of parcels for each of Utah's 29 counties. Each county provides its own data for UGRC, thus, users should expect some variation in schema and completeness between counties.
+
+### What is the purpose of the dataset?
+
+These data represents UGRC's ongoing efforts to compile statewide parcels following the passage of [HB113](https://le.utah.gov/~2005/bills/hbillenr/HB0113.htm) in 2005. Parcel data is used in a variety of contexts, including public safety, economic development, transportation, planning, and the provision of public services.
```

Comment:
```suggestion
These data represents UGRC's ongoing efforts to compile a statewide parcel dataset following the passage of [HB113](https://le.utah.gov/~2005/bills/hbillenr/HB0113.htm) in 2005. Parcel data are used in a variety of contexts, including public safety, economic development, transportation, planning, and the provision of public services.
```

Reply:
mmm, good point. In practice, we treat them as 29 separate layers, but digging into the statute it calls out a single layer and we do create a statewide parcel layer from all the counties (https://opendata.gis.utah.gov/datasets/utah-statewide-parcels/). 

However, for this dataset... @steveoh or @gregbunce, did we decide that this will populate the metadata for each county? Or will this be more for the data page and each county will have it's own metadata addition?